test-cases/terraform/aws/iam/resource-policies/rest_api_not_secure_policy/results_summary.json

```diff
@@ -1,7 +1,7 @@
 {
   "checkov": "no",
   "cloudrail": "yes",
-  "kics": "no",
+  "kics": "yes",
```

Comment:
@kaplanlior and this

Reply:
API Gateway With Open Access

Notice KICS has many warnings while cloudrail doesn't. Maybe cloudrail should be set to no ?

Reply:
This case is also referring to the resource policy.

Reply:
I think the test should be more detailed.

Reply:
More detailed - you mean finding more issues? The original idea in this repo was to have each test case focus on one specific issue, because otherwise it can get messy quickly.

Are there issues KICS catches that are not listed as test cases in the repo? If so, you're more than welcome to add those test cases and test all the tools against them.